packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -41,29 +41,33 @@ Use our typographic styles to communicate visual hierarchy. A consistent and log
   styleProps = {styleProps.tiny} />
 
 ## Line height
-<div style={{marginBottom: '32px'}}>When laying out text-based content, you need to factor in line-height when measuring spacing. Line height is a relative number used in CSS that represents a ratio to the text size. For example, body text uses a line height of 1.5. Body text size is 16px. 16*1.5 = 24px line height. So, when creating designs in design software, you must include the 24px line height as a part of the text.</div>
+<div style={{marginBottom: '32px'}}>Accommodating for line-height is important when measuring spacing in text-based content. 
+
+For example, if a body of text uses a line height of 1.5 and the body text size is 16px, the final line height would be 24 px (16 * 1.5 = 24). In this case, include the 24px line height as part of the text when creating designs in the design software.
+</div>
 
 <Grid>
   <GridItem span={12}>
     <LineHeightTitle>
       <CheckCircleIcon color="#52A549" />
       <span style={{color: '#151515', marginLeft: '8px'}}>Correct</span>
     </LineHeightTitle>
-    <div>Note how the line height space is included when laying out with spacer elements.</div>
+    <div>Note how line height space is considered when laying text with spacer elements.</div>
```

Comment:
Let's make the "do" and "don't" copy consistent. The "don't" copy is more actionable, so let's restructure this "do" copy to align: 

"Include line height space when laying text with spacer elements."